debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -551,6 +551,11 @@ public boolean isFullUpdate() {
             .withValidation(Field::isInteger)
             .withInvisibleRecommender();
 
+    public static final Field MAX_TASKS = Field.createInternal("mongodb.max.tasks")
```

Comment:
@Naros I am a bit confused, why do we need e new config option?

Reply:
Technically yes, but we don't explicitly set `tasks.max` anywhere in the test suite, and it's explicitly passed as a hardcoded value inside of `EmbeddedEngine#run`.  I can most definitely look for `tasks.max` and if not found assume `1` rather than passing the value this way from the task configuration method if you'd prefer.

Reply:
I reworked the first commit to using `tasks.max` instead, defaulting to `1` if the property isn't passed.